File: snippets/go/static_channel_backup.go
package example

import (
	"log"

	"github.com/breez/breez-sdk-go/breez_sdk"
)

func RetrieveBackupFiles() error {
	// ANCHOR: static-channel-backup
	workingDir := "<working directory>"
	staticBackupResponse, err := breez_sdk.StaticBackup(breez_sdk.StaticBackupRequest{WorkingDir: workingDir})
	if err != nil {
		return err
	}
	log.Printf("%#v", staticBackupResponse)
	// ANCHOR_END: static-channel-backup
	return nil
}

File: snippets/python/src/static_channel_backup.py
import breez_sdk

def retrieve_backup_files(sdk_services, working_dir):
    try:
        # ANCHOR: static-channel-backup
        req = breez_sdk.StaticBackupRequest(working_dir=working_dir)
        backup_data = breez_sdk.static_backup(req)
        # ANCHOR_END: static-channel-backup
        return backup_data
    except Exception as error:
        print(error)
        raise